internal/oci/spec_test.go

```diff
@@ -19,14 +19,14 @@ func TestMaintainSpec(t *testing.T) {
 	}
 
 	for _, f := range files {
-		inputSpecPath := filepath.Join(moduleRoot, "test/input", f)
+		inputSpecPath := filepath.Join(moduleRoot, "tests/input", f)
```

Comment:
out of scope for this PR, but we should consider moving this to `testdata`.

Reply:
I'll do as a follow up to this PR